ramls/request-by-instance-id.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "A request for an item based on the specified instance ID",
+  "description": "Request for an item selected from the that might be at a different location or already checked out to another patron",
+  "type": "object",
+  "properties": {
+    "requestDate": {
+      "description": "Date the request was made",
+      "type": "string",
+      "format": "date-time"
+    },
+    "requesterId": {
+      "description": "ID of the user who made the request",
+      "type": "string",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
Is this the stricter UUID regular expression?

I wonder if it is worth adopting that here, to avoid a breaking change later?

Reply:
No, this is the same one used in the other mod-circulation endpoints. I was hesitant to create this one endpoint with a variant regex. If we feel that the more restrict regex is the way to go, than I can update the pattern in this schema.

Reply:
There does not appear to have been an explicit decision made yet. Though the prevailing preference seems to be for stricter.

Given that changing it in other endpoints would be a breaking change, maybe we leave this as is for now.